app/services/foreman/plugin.rb

```diff
@@ -183,7 +183,7 @@ def delete_menu_item(menu, item)
     #                       :name => N_("Example Pagelet"),
     #                       :partial => "path/to/partial",
     #                       :priority => 10000,
-    #                       :id => 'custom-html-id'
+    #                       :hide => Proc.new { |model| model.should_hide_pagelet? }
```

Comment:
I'd suggest inverting this and calling it `onlyif` so that, a) it's positive (avoiding double negatives) and b) matches Rails terminology.